content/en/docs/best-practices/general-principles/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: General Principles
+description: The general principles for your Istio deployments.
+weight: 1
+icon: best-practices
+---
+
+We've identified the following general principles to help you get the most out
+of your Istio deployments.
+
+## Make your deployment simple
+
+When choosing a [deployment model](/docs/concepts/deployment-models),
+**prefer simplicity over complexity** wherever possible.
+
+## Make your deployment fast
+
+Include clusters in your deployment across the globe for **geographic
+proximity to end-users**. The proximity helps your deployment have low latency.
+
+## Make your deployment reliable
+
+Include clusters in your deployment **across multiple availability regions
+and zones** within each region. This approach helps you avoid global failures.
+
+Follow these recommendations to enhance the reliability of your deployment:
+
+- Deploy each workload to at least one cluster per region.
+- Deploy a control plane to at least one cluster per region.
+- Ensure each control plane has a dedicated configuration source.
```

Comment:
This is a bit vague, I am not sure I understand what this means.

In general I think the whole page is a bit high level without explaining how and why to do the things we recommend

Reply:
@rcaballeromx again ... remove this list.